cmd/snap-seccomp-blacklist/Makefile

```diff
@@ -0,0 +1,40 @@
+.PHONY: all
+all: $(foreach v,LE BE,$v-blacklist.bpf) | analyze
+
+.PHONY: clean
+clean:
+	rm -f snap-seccomp-blacklist snap-seccomp-blacklist.o *.pfc *.bpf
+
+.PHONY: fmt
+fmt: snap-seccomp-blacklist.c
+	clang-format -style='{BasedOnStyle: Google, IndentWidth: 4, ColumnLimit: 120}' -i $^
+
+DESTDIR ?=
+
+.PHONY: install
+install:: blacklist.bpf | $(DESTDIR)/var/lib/snapd/seccomp/bpf
```

Comment:
It's my personal preference that I think makes sense for a target such as `install`. It does so because the install target is fulfilled by many separate installations that have their own rules. Since we are not representing the installed file as the goal of the rule using the double colon allows us to have separate entries freely, and not clump them in one complex install rule.